.vscode/settings.json

```diff
@@ -16,5 +16,6 @@
     "--style=.style.yapf"
   ],
   "python.venvPath": "${workspaceFolder}/.venv/",
-  "python.pythonPath": "${workspaceFolder}/.venv/Scripts/python.exe"
+  "python.pythonPath": "${workspaceFolder}/.venv/Scripts/python.exe",
+  "python.unitTest.pyTestEnabled": true
```

Comment:
is true the production value?

Reply:
This setting is to have option to run the test one by one in vscode instead of using the cli, this is just for developement